test/Microsoft.ML.TestFrameworkCommon/Attributes/RetryFactAttribute.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.ML.TestFrameworkCommon.Attributes
+{
+    /// <summary>
+    /// ML.NET facts that will retry several flaky test cases, use default timeout settings
+    /// </summary>
+    [XunitTestCaseDiscoverer("Microsoft.ML.TestFrameworkCommon.RetryFactDiscoverer", "Microsoft.ML.TestFrameworkCommon")]
+    public class RetryFactAttribute : FactAttribute
```

Comment:
Retry Fact is build depending on below sample from xunit:
[https://github.com/xunit/samples.xunit/tree/master/RetryFactExample](https://github.com/xunit/samples.xunit/tree/master/RetryFactExample)